pages/establishment/licence-fees/views/index.jsx

```diff
@@ -91,7 +91,7 @@ export default function Fees({ tab, tabs, children, subtitle = '' }) {
             tabs.map(({ page, key, ...props }, index) => <Link key={index} page={page} label={<Snippet>{key}</Snippet>} {...props} />)
           }
         </Tabs>
-        <div className="tab-content">
```

Comment:
is there other `tab-content` styling? Do we still need the class if not?

Reply:
the class is still there

Reply:
That's why I'm asking, if we've removed the styling do we still need the class name? It's not doing any harm I guess.